### PR TITLE
feat(i18n): account actions

### PR DESCRIPTION
**Pull request by Blazing-Mike**
### Describe the changes you have made in this PR
Adds i18n to account actions.

### Link this PR to an issue
https://github.com/getAlby/lightning-browser-extension/issues/1368

_#ISSUE-NUMBER_

1368

### Type of change (Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### How has this been tested?

tested manually

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/extension/background-script/actions/accounts/decryptedDetails.ts [hunk 1]
```diff
@@ -18,7 +20,10 @@ const decryptedDetails = async (message: MessageAccountDecryptedDetails) => {
     };
   } else {
     return {
-      error: `Account not found: ${accountId}`,
+      error: `${i18n.t(
+        "errors.missing_account",
+        commonI18nNamespace
+      )} ${accountId}`,
```
Comment (escapedcat) [inline src/extension/background-script/actions/accounts/decryptedDetails.ts:26]: I guess you can add the `:` here as well like you did below.

**Comment by Blazing-Mike** [comment]
Ok noted.

**Comment by escapedcat** [comment]
Hey @Blazing-Mike thanks for your PR!  
Unfortunately we decided to not translate the action due to these being mostly errors which the user will never see.
I'll have a look what other issue might be a good one for you.

**Comment by Blazing-Mike** [comment]
No worries, thanks for the update!

**Pull request closed without merging**